src/nvim/api/vim.c

```diff
@@ -695,6 +696,73 @@ Dictionary nvim_get_color_map(void)
   return colors;
 }
 
+// Helper function for nvim_get_keymap and nvim_buf_get_keymap
+// @param mode The abbreviation for the mode
+// @param buffer_id
```

Comment:
This should be `buf_T*` and either NULL or pointer to the buffer. Check API functions in `src/nvim/api/buffer.c`: they all start with getting `buf_T*`.